src/libraries/System.Data.OleDb/src/System.Data.OleDb.csproj

```diff
@@ -3,9 +3,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn);CS1573</NoWarn>
     <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
-    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(OSGroup)' == 'AnyOS' and '$(TargetsNetStandard)' == 'true'">SR.PlatformNotSupported_OleDb</GeneratePlatformNotSupportedAssemblyMessage>
-    <NoWarn Condition="'$(OSGroup)' == 'AnyOS' and '$(TargetsNetStandard)' == 'true'">$(NoWarn);CS0618</NoWarn>
-    <Configurations>net461-Windows_NT-Debug;net461-Windows_NT-Release;$(NetFrameworkCurrent)-Windows_NT-Debug;$(NetFrameworkCurrent)-Windows_NT-Release;netstandard2.0-Debug;netstandard2.0-Release;netstandard2.0-Windows_NT-Debug;netstandard2.0-Windows_NT-Release</Configurations>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetFrameworkSuffix)' == '' and '$(TargetsNetStandard)' == 'true'">SR.PlatformNotSupported_OleDb</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
I'm wondering if we should encode `$(TargetFrameworkSuffix)' == '' into a property, i.e. `TargetsAnyOS`.

Reply:
We could already do it if `OSGroup == AnyOS` just the way we set the other TargetsOS properties.

Reply:
OSGroup is set to the platform on which we are building the project. It will never be equal to AnyOS